treesit-auto.el

```diff
@@ -247,11 +267,42 @@ automatic installation (or prompting, based on the value of
       :url "https://github.com/ikatyang/tree-sitter-markdown"
       :ext "\\.md\\'")
     ,(make-treesit-auto-recipe
-      :lang 'proto
```

Comment:
Issue: It looks like we have an accidental overwrite of the `:lang 'proto` that's required here.